timesketch/lib/utils.py

```diff
@@ -249,11 +368,13 @@ def read_and_validate_redline(file_handle):
         yield row_to_yield
 
 
-def read_and_validate_jsonl(file_handle):
+def read_and_validate_jsonl(file_handle, **kwargs):  # pylint: disable=unused-argument
```

Comment:
It would make more sense. However, since read_and_validate is called here:
```
for event in read_and_validate(
            file_handle=file_handle,
            headers_mapping=headers_mapping,
            delimiter=delimiter
        ):
```

With an if/else statement it would be something like that:
```
if exstension == "csv":
for event in read_and_validate(
            file_handle=file_handle,
            headers_mapping=headers_mapping,
            delimiter=delimiter
        ):
            opensearch.import_event(
                index_name, event_type, event, timeline_id=timeline_id
            )
            final_counter += 1
else:
for event in read_and_validate(file_handle=file_handle):
            opensearch.import_event(
                index_name, event_type, event, timeline_id=timeline_id
            )
            final_counter += 1
```

What do you think?